src/Blorc.OpenIdConnect/Extensions/TypeExtensions.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace Blorc.OpenIdConnect
+{
+    using System;
+
+    public static class TypeExtensions
+    {
+        public static bool IsPrimitiveEx(this Type type)
```

Comment:
make sure to cover this with unit tests (single test using multiple TestCase attributes)